alpha/scripts/utils/restoreMetadataObjects.php

```diff
@@ -0,0 +1,125 @@
+<?php
+ini_set('memory_limit','1024M');
+if ($argc < 3)
+{
+	echo PHP_EOL . ' ---- Restore Deleted Metadata Profile && Objects ---- ' . PHP_EOL;
+	echo ' Execute: php ' . $argv[0] . ' partnerId, metadata profile ID, [ /path/to/metadata object IDs list || metadataObjectId_1,metadataObjectId_2,.. || metadataObject ] [realrun / dryrun]' . PHP_EOL;
```

Comment:
Do you handle path to metadata?
Or only the list of ids?

Reply:
Added handlig to path.